### DISCUSSION TITLE
Welcome to python_ai_toolkit_zos Discussions

### DISCUSSION THREAD

**Discussion by jbostian** [ibm-z-oss-oda/python_ai_toolkit_zos#2] [Announcements]
## 👋 Welcome
This is a primary source of information about the Python AI Toolkit for IBM z/OS.  We will post important information about:

- New versions of packages in the Toolkit
- New packages that are added to the Toolkit
- How-to's and best practices
- Frequently asked questions

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Subscribe to this discussion to be notified of news about the Python AI Toolkit for IBM z/OS.

**Comment by fbh112** [comment]
hi, is https://downloads.pyaitoolkit.ibm.net/repository/python_ai_toolkit_zos/simple still the correct index url? 

I try to install one package from repo, but failed
`pip install pandas  --index-url https://downloads.pyaitoolkit.ibm.net/repository/python_ai_toolkit_zos/simple --trusted-host downloads.pyaitoolkit.ibm.net`
`Looking in indexes: https://downloads.pyaitoolkit.ibm.net/repository/python_ai_toolkit_zos/simple
ERROR: Could not find a version that satisfies the requirement pandas (from versions: none)
ERROR: No matching distribution found for pandas`

**Reply by jbostian** [reply]
Yes, this is the right URL for package downloads.  I've tried this command in my environment, and it works, but only because I have some configuration that a user in a vanilla environment likely won't have.  I'm not in a position right now troubleshoot this, but should be over the next day or so.  I'll update with findings.

**Reply by jbostian** [reply]
@fbh112 - I have scrubbed my pip configuration from my test environment, and tried to re-create this failure several times, but haven't been able to.  Are you still seeing this error?

**Reply by fbh112** [reply]
hi @jbostian I have tired again and managed to install the pandas with **python 3.11**. now it is working fine.

**Reply by anaghagandhi** [reply]
Hi @jbostian , I am getting the same error on Python 3.13. Is there any different setup to be done for 3.13?

ERROR: Could not find a version that satisfies the requirement pandas (from versions: none)
ERROR: No matching distribution found for pandas

**Reply by jbostian** [reply]
Hi @anaghagandhi, unfortunately we don't have Pandas built for Python 3.13 yet.  We are actively working on this package and hope to have it built soon.  If you run into this missing package problem in the future, the first thing you can do is check the appropriate requirements.txt file for the release you're using ([https://github.com/ibm-z-oss-oda/python_ai_toolkit_zos/blob/main/requirements/pyaitoolkit_1.13.0.post3.txt](https://github.com/ibm-z-oss-oda/python_ai_toolkit_zos/blob/main/requirements/pyaitoolkit_1.13.0.post3.txt) in this case).  If the package isn't listed in there, we don't support it yet.
